frontend/src/breeding_form.tsx

```diff
@@ -409,6 +411,7 @@ export function BreedingForm({
         const newBirthData: Birth = {
           date: breeding.birth_date,
           litter_size: breeding.litter_size,
+          litter_size6w: breeding.litter_size,
```

Comment:
The idea is that this value is only interesting when issuing a new certificate or registering a new rabbit with a paper cert. But it would be good to be able to edit it. If it was input wrong. But then you can just issue a new digital certificate with the change I think it is fine for now.